checks/check_status.py

```diff
@@ -625,14 +625,29 @@ def __init__(self, queue_length=0, queue_size=0, flush_count=0, transactions_rec
         self.transactions_received = transactions_received
         self.transactions_flushed = transactions_flushed
 
+        self.proxy_data = get_config().get('proxy_settings', None)
```

Comment:
And you don't need the , None) as it's the default value for the dict.get method.